statistics/histogram.go

```diff
@@ -738,7 +738,7 @@ func (c *Column) equalRowCount(sc *stmtctx.StatementContext, val types.Datum) (f
 		return 0.0, nil
 	}
 	if c.NDV > 0 && c.outOfRange(val) {
```

Comment:
Not related to this PR, just wonder why we need a `c.NDV > 0` check here when we have already checked `c.Histogram.Bounds == nil`.

Reply:
Just to make it clear that it is greater than 0, but actually, we can remove it.